### ISSUE TITLE
Support for batch size more than 1?

### ISSUE THREAD

**Issue by jackjacktang** [KumapowerLIU/Rethinking-Inpainting-MEDFE#6]
Thank you for releasing the code. I noticed that the batch size seems to be fixed to 1, I have tried a larger batch size and it seems not to work due to a size problem in BASE module in PConv, is it possible to support for batch size more than 1?

**Comment by KumapowerLIU** [comment]
Yes, it is can support batch size than 1. But  I am not rewrite the code recently, i will update it as  soon as possible.
